swagger.yaml

```diff
@@ -413,6 +413,30 @@ paths:
           description: "No dimension options were found for dimension"
         500:
           $ref: '#/responses/InternalError'
+  /datasets/{id}/editions/{edition}/versions/{version}/metadata:
+    get:
+      tags:
+      - "Public"
+      summary: "Get all version meta data"
```

Comment:
possibly rather than calling it `version metadata` we should be saying `metadata relevant to a version` as it will contain higher level stuff about the dataset?

Reply:
Also I think metadata is one word